src/template_code.js

```diff
@@ -188,6 +204,15 @@ module.exports = (data, require) => {
     return statusCode >= 500 && statusCode < 600;
   }
 
+  function isStandardEvent(eventName) {
+    for (let i = 0; i < LINE_STANDARD_EVENTS.length; ++i) {
+      if (LINE_STANDARD_EVENTS[i] === eventName) {
+        return true;
+      }
+    }
+    return false;
+  }
+
```

Comment:
Sadly, Sandbox API in Google Server Side Container doesn't support includes method like array in JavaScript...